Gemfile.base

```diff
@@ -91,7 +91,8 @@ group :development, :test do
   end
 end
 
-gem 'nokogiri', '>= 1.6.8'
+gem 'mimemagic', '~> 0.3.10'
```

Comment:
mimemagic is already updated in the PR https://github.com/3scale/porta/pull/2404